src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileService.scala

```diff
@@ -40,11 +39,15 @@ class FileService(
     val now = Timestamp.from(timeSource.now)
     val consignmentId = addFileAndMetadataInput.consignmentId
     val filePaths = addFileAndMetadataInput.metadataInput.map(_.originalPath).toSet
-    val allNodes: Map[String, TreeNode] = treeNodesUtils.generateNodes(filePaths)
+    val allFileNodes: Map[String, TreeNode] = treeNodesUtils.generateNodes(filePaths, fileTypeIdentifier)
+    val allEmptyDirectoryNodes: Map[String, TreeNode] = treeNodesUtils.generateNodes(addFileAndMetadataInput.emptyDirectories.toSet, directoryTypeIdentifier)
+
+    val folderIdTopath: Map[UUID, String] = (allEmptyDirectoryNodes ++ allFileNodes.filter(_._2.treeNodeType == directoryTypeIdentifier))
```

Comment:
Rename to camel case: `folderIdToPath`

Reply:
I knew I'd messed up the case somewhere, I couldn't find it though.